extensions/amp-subscriptions/0.1/viewer-subscription-platform.js

```diff
@@ -122,26 +122,37 @@ export class ViewerSubscriptionPlatform {
       let entitlement = Entitlement.empty('local');
       if (Array.isArray(entitlements)) {
         for (let index = 0; index < entitlements.length; index++) {
-          const entitlementObject =
-              Entitlement.parseFromJson(entitlements[index], token);
-          if (entitlementObject.enables(currentProductId)) {
-            entitlement = entitlementObject;
+          if (entitlements[index].products.indexOf(currentProductId) !== -1) {
```

Comment:
So now you're accessing a json object, right? In that case you need to reference the fields with quotes.